php-8.1-opentelemetry.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: php-8.1-opentelemetry
+  version: 1.0.1
+  epoch: 0
+  description: "OpenTelemetry PHP auto-instrumentation extension"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ${{package.name}}-config
+      - php-8.1
+    provides:
+      - php-opentelemetry=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - build-base
+      - busybox
+      - php-8.1
+      - php-8.1-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-telemetry/opentelemetry-php-instrumentation
+      tag: ${{package.version}}
+      expected-commit: 7010de909f8fa65ae83ffe3ffa815c82c71842c6
+
+  - name: Prepare build
+    runs: cd ext && phpize
```

Comment:
Is there something general that I can set, kinda `sourceRoot=ext` ?. Didn't found anything in the docs

Reply:
you can use working-directory 

```
    pipeline:
      - working-directory: /home/build/ext
 ```

There are example son openjdk and Kubernetes of this